src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -56,13 +35,7 @@ internal sealed class EventPipeController
             new EventPipeProviderConfiguration("Microsoft-DotNETCore-SampleProfiler", 0x0, 5, null),
         };
 
-        // Singleton controller instance.
-        private static EventPipeController? s_controllerInstance;
-
-        // Controller object state.
-        private Timer m_timer;
-        private string m_configFilePath;
-        private bool m_configFileExists;
+        private static bool IsControllerInitialized { get; set; } = false;
 
         // Initialization flag used to avoid initializing FrameworkEventSource on the startup path.
         internal static bool Initializing { get; private set; }
```

Comment:
You should be able to remove ```Initializing``` as well.  It was used to avoid initializing FrameworkEventSource as part of Timer creation on the start-up path.  Now that the Timer creation code is being removed, this can be removed as well along with its use in the Timer implementation.